### ISSUE TITLE
linux-browser update

### ISSUE THREAD

**Issue by hakova** [mrclksr/linux-browser-installer#36]
Hi,

I am sorry if this is a silly question. What is the best way to update the installed browser? I am worried that simply deinstalling and reinstalling the browser would remove the bookmarks and customized toolbar settings.

Thank you!

**Comment by mrclksr** [comment]
Hi @hakova,

run `./linux-browser-installer chroot upgrade`. The script does not modify any files in the user's directory, so your settings and bookmarks are safe. If unsure, you can back them up, e.g.:

```
% cp -a ~/.config/google-chrome dot.config.google-chrome
% cp -a ~/.config/BraveSoftware dot.config.BraveSoftware
...
```

**Comment by hakova** [comment]
Thank you so very much @mrclksr, both for making this wonderful tool available and your quick reply! It works wonderful for me and is rock stable! Very much appreciated!

**Comment by hakova** [comment]
I am sorry I had to reopen this issue because upgrading the chroot totally broke everything. Chrome did not launch after the upgrade. I tried launching it from the terminal and got the following error:
`ELF interpreter /lib64/ld-linux-x86-64.so.2 not found, error 2`
Then I followed the recommendations on issue #27 and attempted to delete and create chroot as recommended there both with focal and kinetic settings in the script.
chroot create results in the following error now:
`Error: debootstrap failed`
With a desperate attempt, I tried `rm -rf /compat` as root, which curiously also failed. I am totally lost now. I shouldn't have upgraded it, now I don't have a working system anymore and I am not sure how to fix it.
